plugin/src/main/scala/io/github/davidgregory084/ScalaVersion.scala

```diff
@@ -21,6 +21,8 @@ import scala.Ordering.Implicits._
 case class ScalaVersion(major: Long, minor: Long, patch: Long) {
   def isBetween(addedVersion: ScalaVersion, removedVersion: ScalaVersion) =
     this >= addedVersion && this < removedVersion
+
+  def isAfter(addedVersion: ScalaVersion) = this >= addedVersion
```

Comment:
Since it's non-strict? just bikeshed

```suggestion
  def isOrAfter(addedVersion: ScalaVersion) = this >= addedVersion
```

Reply:
`isAtLeast`?